middlewares/validators/progresses.js

```diff
@@ -92,8 +92,26 @@ const validateGetRangeProgressRecordsParams = async (req, res, next) => {
     res.boom.badRequest(error.details[0].message);
   }
 };
+
+const validateGetDayProgressParams = async (req, res, next) => {
+  const schema = joi.object({
+    type: joi.string().valid("user", "task").required().messages({
```

Comment:
Isn't separating these types into a separate Array do better. Incase we need to add more entities. Or, are you sure there wont be any more types in future?